python_examples/Gibraltar_example/gibraltar_inputs/microcell_conversion.py

```diff
@@ -0,0 +1,73 @@
+#
+# Converts gibraltar population data from covidsim into compatible input file
+#
```

Comment:
If this a duplicated file?

Reply:
yes at the moment. Wanted to keep all folders self contained as am anticipating deleting the covdisim comparisonfolder

Reply:
Yep that makes sense!